### ISSUE TITLE
Need to escape single quotes (') in xconfig.sh

### ISSUE THREAD

**Issue by alllexx88** [arsv/perl-cross#19]
Today, I stumbled upon a very odd compilation error while using perl-cross:

    ./xconfig.sh: line 222: syntax error near unexpected token `('
    ./xconfig.sh: line 222: `d_Gconvert='sprintf((b),"%.*g",(n),(x))''

After some investigation, I discovered that the following line was the culprit:

    cf_time='п'ятниця, 30 жовтня 2015 17:17:54 +0200'

The value of cf_time is obviously taken from `date` output, and in Ukrainian "Friday" is "п'ятниця", hence that <'> symbol in date string wreaks havoc to the script. I simply changed date language (`LC_TIME`) in regional formats, re-logined, re-configured perl-cross, and it all built fine.

To sum up, my point is that it makes sense to either invoke `date` with `LC_TIME=en_US.UTF-8`, or, better, to escape single quotes when generating xconfig.sh

**Comment by arsv** [comment]
Och, yeah, thanks for noticing.

Resetting locale for date looks like a better solution to me.
Apparently the only place it's used in is perl -V output which is in English regardless of locale settings, so it makes sense to keep it all in English.

**Comment by alllexx88** [comment]
Agree, that makes sense. I take it, no other variables can (even in theory) have <'> in them, right? It's just that if they happen to, this error is very misleading, I spent some time trying to figure it out...

**Comment by arsv** [comment]
Actually a lot of them can.
There's nothing preventing the user from doing something like -Dcf_by="O'Brien".

Added proper quoting, it's ugly but it's probably better than dealing with stray quotes.

**Comment by alllexx88** [comment]
I think this fixes it, thanks
